### PR TITLE
Mariana legacy css

### LINKED ISSUE #550: Add News & Announcement Components
The news component currently on the homepage and a few other pages needs to be added into patterns. 
https://d8-uat.boston.gov/departments/digital-team/test-page
https://d8-uat.boston.gov/

Hover and defaults states

image

Suggested naming conventions:
   Base:
    news or n--a

   Elements:
    h = header
    c = content
    im = image
    t = title
    st = subtitle

   Modifier:
    sm = small

### LINKED ISSUE #553: GRID OF QUOTES
https://d8-uat.boston.gov/departments/digital-team/test-page
GRID OF QUOTES

Screen Shot 2021-06-30 at 4 53 48 PM

Suggested naming conventions:
Base:
goq

Elements:
h = header
c = content
im = image
t = title
st = subtitle

Modifier:
sm = small

### LINKED ISSUE #559: Date Flag
Date Flag
https://d8-uat.boston.gov/departments/digital-team/test-page

Screen Shot 2021-07-08 at 3 14 19 PM

This is base on the output from this page on [ZeroHeight](https://zeroheight.com/7316f2c4a/p/192f88-flags/b/59bc45)

### PR DESCRIPTION
**Pull request by mharoc**
Development Branch for the following issues:
- #559
- #550
- #553

### CODE REVIEW

File: components/base/grid of quotes/gridofquotes.hbs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            </div>
+              </div>
+      </div>
+</div>
+
+          </div>
+
+    </div>
+
+  </div>
```
Comment (phillipbentonkelly) [inline components/base/grid of quotes/gridofquotes.hbs]: End every file with an empty new line (\n).

**Review by phillipbentonkelly — CHANGES_REQUESTED** [review]
Please apply the following changes
- The Quotes template needs work, the scaling and formatting doesn't match the component yet. Once you have it looking right add another with different text, text length to show what it looks like with longer text than expected
  - make the icon's default/max dimensions smaller
- rename the `quote` directory to exclude the spaces between, linking to [google's naming conventions](https://developers.google.com/style/filenames#:~:text=to%20file%20types-,Naming%20guidelines,in%20file%20and%20directory%20names.) as a good start/guide in general
- Add more flags, pref the other variants shown in [zero heights](https://zeroheight.com/7316f2c4a/p/192f88-flags/b/59bc45) excluding the `ico` one
- Use a formatter to make the code (specially the HTML/Handlebars templates) more readable. Prettier is the most used formatter, though I don't think works for these template but its works having for everything else Screen Shot 2021-07-30 at 11 41 05 AM
- End files in with a new (empty) line (\n)
- Is the work for #555 in this branch?
- The news component needs work, when you view it as `Component only` the formatting break. We can discuss bc I am not sure what the directions were here.

**Review by phillipbentonkelly — CHANGES_REQUESTED** [review]
In Draft mode for Stella to take a look at again

**Review by subaha-cob — APPROVED** [review]
LGTM

**Pull request merged**